### PR TITLE
Setup desks - release/v2.0

### PR DESCRIPTION
**Pull request by PatrykJurczyk**

### CODE REVIEW

File: billennium/apps/billespace/src/app/modules/office-management/components/setup-desks.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  gap: 20px;
+`;
+
+const Wrapper = styled.div`
+  margin-top: 128px;
+`;
+
+const Footer = styled.footer`
+  text-align: right;
+  ${Button} {
```
Comment (polubis) [inline billennium/apps/billespace/src/app/modules/office-management/components/setup-desks.tsx:20]: Add enter

**Review by polubis — CHANGES_REQUESTED** [review]
LGTM - small styling stuff and -> please squash commits and follow conventions

File: billennium/apps/billespace/src/app/modules/office-management/components/setup-desks.tsx (hunk 1 above)
Comment (PatrykJurczyk) [reply billennium/apps/billespace/src/app/modules/office-management/components/setup-desks.tsx:20]: Done

**Review by polubis — APPROVED** [review]
LGTM merged

**Pull request merged**